fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs

```diff
@@ -765,6 +765,110 @@ human-readable data."
       deprecated = NotDeprecated }
 
 
+    { name = fn "DarkInternal" "startStaticAssetDeploy" 0
+      parameters = [ Param.make "username" TStr ""; Param.make "canvasID" TUuid "" ]
+      returnType =
+        TResult(
+          TRecord [ "deployHash", TStr
+                    "url", TStr
+                    "status", TStr
+                    "lastUpdate", TDate ],
+          TStr
+        )
+      description = "Records an in-progress static asset deployment"
+      fn =
+        internalFn (function
+          | _, [ DStr username; DUuid canvasID ] ->
+            uply {
+              match! Account.getUser (UserName.create username) with
+              | None ->
+                Exception.raiseInternal $"User not found" [ "username", username ]
+                return DResult(Error(DStr "User not found"))
```

Comment:
You don't need this if you're raising an exception. Where do you want the error checking to go here? If you're planning on returning a `DResult` then you don't need an exception, but if you're planning on a `DError`, then what is the point in `DResult`